submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,17 @@
+### Complexity Corner: The Mall <author>The Kool</author>
+
+Welcome back once more to the Complexity Corner.  Today I'll be exploring a technique instead of a technology: the Mall.  As always with these articles, we'll be asking three questions: 'What is it?', 'What results can it give?', and 'Where should it be used?'
```

Comment:
Since this is a shorter article, could you maybe include something about the origins of the mall? Maybe who came up with it first and what theirs looked like? It might not be possible to find the information but I think it could flesh out the piece if possible.